portal/src/ScreenHeader.tsx

```diff
@@ -173,6 +175,12 @@ interface ScreenNavProps {
   showHamburger?: boolean;
 }
 
+registerIcons({
+  icons: {
+    Cookies: <Cookies16Regular />,
+  },
+});
```

Comment:
Please move this call of registerIcons to src/index.tsx. By convention, we do all initialization in that file.

Reply:
Updated